book/snippets/14-MVC.es2015.js

```diff
@@ -0,0 +1,141 @@
+
+//*******************************************************//
+// Models
+//*******************************************************//
+
+//********************** Snippet 1 **********************//
+var Photo = Backbone.Model.extend({
+ 
+  // Default attributes for the photo
+  defaults: {
+    src: "placeholder.jpg",
+    caption: "A default image",
+    viewed: false
+  },
+
+  // Ensure that each photo created has an `src`.
+  initialize: function() {
+     this.set( { "src": this.defaults.src} );
+  }
+
```

Comment:
I have a passing comment on code-style. For many of the ES2015 versions, we should opt for the shorter arrow function syntax where possible and consider var -> const/let where it makes sense. As a sanity check, you can try running the snippet through http://lebab.io/try-it to evaluate whether a more terse form of snippet could be possible.

```js
const Photo = Backbone.Model.extend({
 
  // Default attributes for the photo
  defaults: {
    src: "placeholder.jpg",
    caption: "A default image",
    viewed: false
  },

  // Ensure that each photo created has an `src`.
  initialize() {
     this.set( { "src": this.defaults.src} );
  }
```
});

Reply:
Yes, that's a reasonable comment. But this is a case that I told you in chat about Backbone and their documentation. But if you think so I could change that code to es2015 too.